Misc/NEWS.d/next/Core and Builtins/2020-11-13-17-25-44.bpo-42349.JdWxez.rst

```diff
@@ -0,0 +1,2 @@
+* Make sure that the compiler front-end produces a well-formed control flow graph.
+* Be more aggressive in the compiler back-end, as it is now safe to do so.
```

Comment:
I think we can not use lists in a news entry.

Reply:
Newlines in news entries are 'soft' so that each is re-wrapped to line lengths of (on my screen) 5 1/2 to 9 inches.  (Shorter screen space get a horizontal scrollbar, longer has the extra space ignored.  The *s are not needed here anyway.

Reply:
Thanks for pointing that out. Fixed.